tests_rv/patch/build_warn_rv64/build_warn_rv64.sh

```diff
@@ -22,10 +23,18 @@ git log -1 --pretty='%h ("%s")' HEAD~
 echo "Baseline building the tree"
 
 tuxmake --wrapper ccache --target-arch riscv -e PATH=$PATH --directory . \
-	--environment=KBUILD_BUILD_TIMESTAMP=@1621270510 \
+	--environment=KBUILD_BUILD_TIMESTAMP=@1621270510 --fail-fast \
 	--environment=KBUILD_BUILD_USER=tuxmake --environment=KBUILD_BUILD_HOST=tuxmake \
 	-o $tmpdir0 --toolchain gcc -z none --kconfig allmodconfig -K CONFIG_WERROR=n \
-	-K CONFIG_GCC_PLUGINS=n W=1
+	-K CONFIG_GCC_PLUGINS=n \
+	> $tmpfile_b || rc=1
+
+if [ $rc -eq 1 ]
+then
+	echo "Failed to build the tree with this patch." >&$DESC_FD
+	grep "\(warning\|error\):" $tmpfile_b >&2
+	exit $rc
+fi
 
```

Comment:
As previously stated, this is a leftover from NIPA, and can be removed (i.e. the whole extra "baseline" build).